default/scripting/starting_unlocks/items.inf

```diff
@@ -12,20 +12,24 @@ Item type = Building name = "BLD_SHIPYARD_ORBITAL_DRYDOCK"
 Item type = Building name = "BLD_EVACUATION"
 Item type = Building name = "BLD_COL_SUPER_TEST"
 Item type = Building name = "BLD_COL_ABADDONI"
+Item type = Building name = "BLD_COL_BANFORO"
 Item type = Building name = "BLD_COL_CHATO"
 Item type = Building name = "BLD_COL_CRAY"
 Item type = Building name = "BLD_COL_DERTHREAN"
 Item type = Building name = "BLD_COL_EAXAW"
 Item type = Building name = "BLD_COL_EGASSEM"
 Item type = Building name = "BLD_COL_ETTY"
+Item type = Building name = "BLD_COL_EXOBOT"
```

Comment:
Is that still needed here? If the Exobots are the big exception, you should only be able to build their colony building if you have the tech, so no need to pre-unlock it (unless I'm missing something).